task/common/machine/storage.go

```diff
@@ -120,6 +124,12 @@ func Transfer(ctx context.Context, source, destination string, include string) e
 		return err
 	}
 
+	if count, size, err := operations.Count(ctx, sourceFileSystem); err == nil {
+		logrus.Infof("Transferring %d files with a total size of %s; this may take a while", count, units.HumanSize(float64(size)))
+	} else {
+		return err
+	}
+
 	return sync.CopyDir(ctx, destinationFileSystem, sourceFileSystem, true)
```

Comment:
```suggestion
	if err := sync.CopyDir(ctx, destinationFileSystem, sourceFileSystem, true); err != nil {
		return err
	}
	
	logrus.Info("Transferring done")
	return nil
```

Reply:
@casperdcl, this message isn't particularly useful, as all the operations are performed serially and the next message signals that the previous operation has finished.

Reply:
No strong opinions, but I would leave it as–is for consistency with the other operations.

Reply:
> all the operations are performed serially and the next message signals that the previous operation has finished

Fine by me to omit message here if we really stick to this convention (i.e. zero chance of error before next message is logged)

Reply:
> _zero chance of error_

In computers? Cough.

> _zero chance of error before next message is logged_

If there is an error, we'll print it anyway and abort the process.